tasks/src/main/java/com/example/task/DeleteQueue.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.example.task;
+
+// [START cloud_tasks_delete_queue]
+import com.google.cloud.tasks.v2.CloudTasksClient;
+import com.google.cloud.tasks.v2.QueueName;
+
+public class DeleteQueue {
+  /**
+   * Delete a queue using the Cloud Tasks client.
+   *
+   * @param projectId the Id of the project.
+   * @param queueId the name of your Queue.
+   * @param locationId the GCP region of your queue.
+   * @throws Exception on Cloud Tasks Client errors.
+   */
+  public static void deleteQueue(String projectId, String locationId, String queueId)
+      throws Exception {
```

Comment:
Ideally we are as specific as possible with exceptions based on what developers might see.